cmd/score-compose/main.go

```diff
@@ -15,8 +15,11 @@ import (
 )
 
 func main() {
-	if err := command.Execute(); err != nil {
-		fmt.Fprintln(os.Stderr, err)
-		os.Exit(1)
+	status := command.ParseOpts()
```

Comment:
I would recommend we flip this around so that we can rely on the more battle-tested parser (cobra) first and then if there's an error there, we can attempt to add additional information using the custom parsing run.

For example:

1. Run command.Execute()
2. If there's no error, return
3. If there is an error, and it's a CLI error from cobra then we run the custom parser and attempt to add additional information

In this way, we always fallback to a safe CLI behavior.